docs/explanation/chisel.rst

```diff
@@ -25,7 +25,7 @@ slices.
 The use of package slices provides Rockcraft with the ability to build minimal
 container images from the wider set of Ubuntu packages.
 
-.. figure:: /_static/package-slices.svg
+.. figure:: ../_static/package-slices.svg
```

Comment:
why the `../`? The existing form sounds safer cause it's an absolute path. If we move this chisel.rst file tomorrow, this will break